NitroxPatcher/Patches/Dynamic/DayNightCycle_OnConsoleCommand_daynightspeed_Patch.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Reflection;
+using HarmonyLib;
+using NitroxClient.Communication.Abstract;
+using NitroxModel.Helper;
+using NitroxModel.Packets;
+
+namespace NitroxPatcher.Patches.Dynamic
+{
+    class DayNightCycle_OnConsoleCommand_daynightspeed_Patch : NitroxPatch, IDynamicPatch
+    {
+        public static readonly MethodInfo TARGET_METHOD = Reflect.Method((DayNightCycle t) => t.OnConsoleCommand_daynightspeed(default(NotificationCenter.Notification)));
+        private static IPacketSender packetSender;
+
+        public static bool Prefix(NotificationCenter.Notification n)
```

Comment:
It should be explained here that we skip the original method because the server should control the speed and not the client.